go.mod

```diff
@@ -1,26 +1,29 @@
 module github.com/cloudogu/k8s-service-discovery
 
-go 1.21
+go 1.22.0
+
+toolchain go1.22.5
 
 require (
 	github.com/bombsimon/logrusr/v2 v2.0.1
 	github.com/cloudogu/cesapp-lib v0.12.2
 	github.com/cloudogu/k8s-dogu-operator v0.38.0
+	github.com/cloudogu/k8s-registry-lib v0.0.0-20240808134819-4a76d5bcf07f
```

Comment:
`v0.3.0` is released now, so you can update. There's quite a few other dependencies that could be updated as well, for example with `go get -u`.